_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/WordpressApi.php

```diff
@@ -235,6 +242,12 @@ public function getCategoryOptions() {
     return $categories;
   }
 
+  /**
+   * Please update this function description..
+   *
+   * @return array
+   *   Please update this return description.
+   */
   private function getContentComposite($content) {
```

Comment:
This param is an object of \stdClass() that was created from `json_decode()`ing the response from the wp-json API. You can see lines 52-55 of this file to see how this works.